app/src/main/java/com/keylesspalace/tusky/components/viewthread/ViewThreadViewModel.kt

```diff
@@ -107,7 +107,7 @@ class ViewThreadViewModel @Inject constructor(
             val contextCall = async { api.statusContext(id) }
             val timelineStatus = db.timelineDao().getStatus(id)
 
-            var detailedStatus = if (timelineStatus != null) {
+            var detailedStatus = if (timelineStatus != null && !timelineStatus.status.isPlaceholder) {
```

Comment:
```suggestion
            // TODO: Investigate how this could ever be the ID of a placeholder status
            var detailedStatus = if (timelineStatus != null && !timelineStatus.status.isPlaceholder) {
```

The change LGTM, at least as far as preventing the crash goes.

Can you reproduce this at will? From looking at the code I can't immediately see a path that would ever result in the ID passed to this function being from a placeholder status. AIUI, the user only gets here by some route that involves clicking on a concrete status.

If you can reproduce it, and can provide reproduction instructions, I can also work on a deeper fix (which shouldn't prevent this PR from being merged in the meantime).